benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -477,6 +68,9 @@ class CmdLineArgs
   char **argv() { return _argv.data(); }
 };
 
+extern void register_bvh_benchmarks(Spec const &spec);
+extern void register_boostrtree_benchmarks(Spec const &spec);
```

Comment:
Yuk

Reply:
I thought about it more and I think that is the very thing that I dislike about the way you structured the benchmark.
There is something messed up about the dependencies.  I would prefer a static vector to register functions that you loop through in that file (push back in the `benchmarks_foo.cpp` files).

That said I am not blocking.

Reply:
I shared https://github.com/dalg24/ArborX/commit/f332086bfeada3e0802ad50757954a075d733a10 on Slack